resource/index.go

```diff
@@ -55,33 +55,31 @@ func (r *index) Compile() error {
 //
 // If a parent is given and the path starts with a dot, the lookup is started at the
 // parent's location instead of root's.
-func (r *index) GetResource(path string, parent *Resource) (*Resource, bool) {
+func (r *index) GetResource(path string, parent *Resource) (sr *Resource, found bool) {
```

Comment:
I actually find the explicit double return more explicit and readable.

Although Go allows named returns, it may be error prone some times, especially with multiple return values. As it's less explicit, it's easier to (by mistake) return a pair of values that was not meant to be returned, and it's harder to read the code to find out which pair of variables are returned in different conditions.

 I tend to only use named returned parameters when it's necessary for technical reasons, such as setting an error in a deferred recover. The name can also serve as documentation when it's in the header, but even there it's only really needed if the semantic meaning of the return values can not be easily deduced from the method name, return types and go conventions.

Sorry for shooting this down, I just have done to a few to many mistakes myself with named return values when I started using Go (2010/2011), and they are still fresh in memory:-)